lib/chains/llm_chain.ex

```diff
@@ -226,6 +226,18 @@ defmodule LangChain.Chains.LLMChain do
 
         {:ok, updated_chain}
 
+      # When calling the latest OpenAI models through Azure it sometimes returns an empty list
+      # as the first element of a list of MessageDeltas. This is a workaround to handle that.
+      # It would likely be better to handle this in ChatOpenAI instead.
+      {:ok, [[] | _] = deltas} ->
```

Comment:
Yes, this peculiarity should be handled in the LLM module instead. It would also need a test case. 

Perhaps we can pull this out into a separate PR?